docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,50 @@
+        - otherwise, rule is not applicable: `else: RULE NOT APPLICABLE`
+
+            ## Rule Logic:  
+                - 9.5.2.2(b) gives a formula (750 W + (Retail Area 1 × 0.40 W/ft2) + (Retail Area 2 × 0.40 W/ft2) + (Retail Area 3 × 0.70 W/ft2) + (Retail Area 4 × 1.00 W/ft2)) for retail display lighting that is based on four area categories.  We don't have access to these four area categories in the schema, so we will calculate the maximum and minimum values possible based on this function.  The maximum is calculated based on 100% of the space floor area being type 4: `maximum_retail_display_W = 750 + space.floor_area * (1)`
+                - the minimum is calculated based on none of the floor area being a retail area: `minimum_retail_display_W = 750`
+                - now calculate the total proposed interior display lighting power for this space - initialize at 0: `proposed_interior_display_W = 0`
+                - look at each lighting in the space: `for interior_lighting in space.interior_lighting:`
+                    - if the interior lighting purpose_type is RETAIL_DISPLAY, add the lighting wattage to proposed_interior_display_W: `if interior_lighting.purpose_type == "RETAIL_DISPLAY": proposed_interior_display_W = proposed_interior_display_W + interior_lighting.power_per_area * space.floor_area`
+                - get the equivalent space in the baseline model: `space_b = get_component_by_id(space.id, B_RMD);`
+                - calculate to total baseline interior display lighting power for this space - initialize at 0: `baseline_interior_display_W = 0`
+                - look at each lighting in the space: `for interior_lighting in space_b.interior_lighting:`
+                    - if the interior lighting purpose_type is RETAIL_DISPLAY, add the lighting wattage to baseline_interior_display_W: `if interior_lighting.purpose_type == "RETAIL_DISPLAY": baseline_interior_display_W = baseline_interior_display_W + interior_lighting.power_per_area * space_b.floor_area`
+
+                **Rule Assertion:**
+                - Case 1: If the proposed_interior_display_W is less than the minimum AND the baseline_interior_display_W is equal to the proposed, then PASS: `if((proposed_interior_display_W < minimum_retail_display_W) and (baseline_interior_display_W == proposed_interior_display_W)): PASS`
+                - Case 2: Otherwise, if the baseline_interior_display_W is greater than the maximum of proposed_interior_display_W and maximum_retail_display_W, then FAIL: `elif baseline_interior_display_W > max(proposed_interior_display_W,maximum_retail_display_W): FAIL`
+                - Case 3: All other cases UNDETERMINED and provide note: `else: UNDETERMINED; note = "The RCT could not determine whether the baseline retail display lighting power is correctly the minimum of the proposed retail display lighting power and the result of the formula given by ASRAE 90.1 9.5.2.2(b)."`
```

Comment:
Suggested note : "It could not be determined whether the baseline retail display lighting power is modeled correctly as the minimum of the proposed retail display lighting power and the allowance calculated according to the formulas in ASHRAE 90.1 Section 9.5.2.2(b)."